cmd/plotFunctions/plotFunctions.go

```diff
@@ -90,11 +93,12 @@ func main() {
 		outFile := flag.Arg(2)
 		words := strings.Split(functionArgs, ",")
 		if len(words) != 2 {
-			log.Fatalf("An allele frequency probability mass function is defined by two parameters, received %d.", len(words))
+			log.Fatalf("An allele frequency probability mass function is defined by three parameters, received %d.", len(words))
 		}
 		alpha := common.StringToFloat64(words[0])
 		n := common.StringToInt(words[1])
-		popgen.PlotAfsPmf(alpha, n, outFile)
+		integralError := common.StringToFloat64(words[2])
+		popgen.PlotAfsPmf(alpha, n, outFile, integralError)
 	} else if flag.Arg(0) == "ChooseN" {
```

Comment:
This is where a switch really shines
```
switch flag.Arg(0) { 
case "ChooseN": ...
case "AfsF": ...
case "AfsFCareless": ...
...
}
```
Not requesting the change, but just saying...